### ISSUE TITLE
css: change styling of SignUp button

### ISSUE THREAD

**Issue by akshayitzme** [akshayitzme/cinlist#25]
currently, we haven't styled SignUp button in the  [index page](http://localhost:8000/). please write necessary styles to make the button look good.
![image](https://user-images.githubusercontent.com/60477442/194523464-ca85925f-6da9-4da5-9c91-6486ed2dc8e4.png)

- Target File: [style.css](https://github.com/akshayitzme/cinlist/blob/master/cinlist/static/css/style.css)
- ✨ Feel free to add your own changes ! 
- 💬 drop a comment if you like to take up this issue, so i can assign it to you.

**Example:**
```css
...

/* Register Button */

.btn-register {
    background: var(--primary);
    border: none;
    padding: 10px 45px;
    border-radius: 2px;
    font-size: 22px;
    color: #fff;
    font-weight: 500;
}
```

**Comment by heraldjose10** [comment]
I would like to take this up. Can I style it the way I want?

**Comment by akshayitzme** [comment]
> I would like to take this up. Can I style it the way I want?

sure ! go ahead

**Comment by heraldjose10** [comment]
Completed. Take a look at PR #36 . Thanks!

**Comment by akshayitzme** [comment]
fixed in #36
